PocketKit/Sources/Analytics/Tracker/PocketTracker.swift

```diff
@@ -30,9 +31,10 @@ public class PocketTracker: Tracker {
         snowplow.track(event: event)
     }
 
-    public func track(event: Event) {
+    public func track(event: Event, filename: String = #file, line: Int = #line, column: Int = #column, funcName: String = #function) {
         let selfDescribing = event.toSelfDescribing()
         persistentContexts.forEach { selfDescribing.contexts.add($0) }
```

Comment:
does this call also crash for you? It crashes since I believe persistentContexts includes the deprecated entities as opposed to using the new ones. 

Getting clarity on whether we still want this going forward: https://pocket.slack.com/archives/CQKRXK2G6/p1676990294127479?thread_ts=1676653246.345029&cid=CQKRXK2G6

Reply:
Cool thank you!